cups/file.c

```diff
@@ -704,6 +704,160 @@ cupsFileGetChar(cups_file_t *fp)	/* I - CUPS file */
   return (*(fp->ptr)++ & 255);
 }
 
+/*
+ * 'cupsFileGetConfAndComments()' - Get a line and Comments from a configuration file.
+ */
+
+char *					/* O  - Line read or @code NULL@ on end of file or error */
+_cupsFileGetConfAndComments(
+		char *comment,		/* Comment */
```

Comment:
Fix indentation - 4 spaces when the arguments can't be fully indented and line up all variable names.

Also, you need to add a "commentlen" argument to specify the size of the comment buffer.

Finally, "I" prefix for input variables, "O" for output, and "IO" for input + output (so "I" for comment and commentlen, "IO" for comment_line)